specs/enhetsregisteret.json

```diff
@@ -161,14 +168,147 @@
+        {
+          "name": "registrertIStiftelsesregisteret",
+          "in": "query",
+          "description": "Hvorvidt enheten er registrert i Stiftelsesregisteret",
+          "required": false,
+          "schema": {
+            "type": "boolean"
+          }
+        },
+        {
+          "name": "registrertIFrivillighetsregisteret",
+          "in": "query",
+          "description": "Hvorvidt enheten er registrert i Frivillighetsregisteret",
+          "required": false,
+          "schema": {
+            "type": "boolean"
+          }
+        },
+        {
+          "name": "frivilligRegistrertIMvaregisteret",
+          "in": "query",
+          "description": "Frivillig registrert i Merverdiavgiftsregisteret. Kommaseparert liste med beskrivelser.",
+          "required": false,
+          "schema": {
+            "type": "string"
+          }
+        },
+        {
+          "name": "underTvangsavviklingEllerTvangsopplosning",
+          "in": "query",
+          "description": "Hvorvidt enheten er registrert som underTvangsavvikling eller tvangsopplosning",
+          "required": false,
+          "schema": {
+            "type": "boolean"
+          }
+        },
+        {
+          "name": "underAvvikling",
+          "in": "query",
+          "description": "Hvorvidt enheten er registrert som underAvvikling",
+          "required": false,
+          "schema": {
+            "type": "boolean"
+          }
+        },
+        {
+          "name": "fraStiftelsesdato",
+          "in": "query",
+          "description": "Tidligste stiftelsesdato for enheten",
+          "required": false,
+          "schema": {
+            "type": "string",
+            "format": "date (ISO-8601)"
+          }
+        },
+        {
+          "name": "tilStiftelsesdato",
+          "in": "query",
+          "description": "Seneste stiftelsesdato hos enheten",
+          "required": false,
+          "schema": {
+            "type": "string",
+            "format": "date (ISO-8601)"
+          }
+        },
+        {
+          "name": "hjemmeside",
+          "in": "query",
+          "description": "Enhetens hjemmeside",
+          "required": false,
+          "schema": {
+            "type": "string"
+          }
+        },
+        {
+          "name": "institusjonellSektorkode",
+          "in": "query",
+          "description": "Enhetens institusjonelle sektorkode. Kommaseparert liste med sektorkoder på 4 siffer.",
+          "required": false,
+          "schema": {
+            "type": "string"
+          }
+        },
+        {
+          "name": "postadresse",
+          "in": "query",
+          "required": false,
+          "schema": {
+            "$ref": "#/components/schemas/Adresse"
+          }
+        },
+        {
+          "name": "forretningsadresse",
+          "in": "query",
+          "required": false,
+          "schema": {
+            "$ref": "#/components/schemas/Adresse"
+          }
+        },
```

Comment:
postadresse og forretningsadresse er ikke støttet som standalone-query-parametre. Disse krever ytterligere spesifisering, f.eks. postadresse.postnummer. Merk at søkeparameteret kommunenummer eksisterer i stedet for forretningsadresse.kommunenummer, mens resterende filtre på forretningsadresse krever prefiksen forretningsadresse.

Reply:
Det gir mening. Skal fikse.

Reply:
Da var endringene gjort 2463c7929c8b031f8ae542d81f5e8a714c684239